src/main/java/com/code_intelligence/jazzer/mutation/mutator/lang/IntegralMutatorFactory.java

```diff
@@ -112,6 +112,39 @@ public void write(Short value, DataOutputStream out) throws IOException {
               out.writeShort(value);
             }
           });
+    } else if (clazz == char.class || clazz == Character.class) {
+      return Optional.of(
+          new AbstractIntegralMutator<Character>(type, Character.MIN_VALUE, Character.MAX_VALUE) {
+            @Override
+            protected long mutateWithLibFuzzer(long value) {
+              return LibFuzzerMutate.mutateDefault((char) value, this, 0);
+            }
+
+            @Override
+            public Character init(PseudoRandom prng) {
+              return (char) initImpl(prng);
+            }
+
+            @Override
+            public Character mutate(Character value, PseudoRandom prng) {
+              return (char) mutateImpl(value, prng);
+            }
+
+            @Override
+            public Character crossOver(Character value, Character otherValue, PseudoRandom prng) {
+              return (char) crossOverImpl(value, otherValue, prng);
+            }
+
+            @Override
+            public Character read(DataInputStream in) throws IOException {
+              return (char) forceInRange(in.readChar());
+            }
+
+            @Override
+            public void write(Character value, DataOutputStream out) throws IOException {
+              out.writeShort(value);
```

Comment:
Out of curiosity, why not use `writeChar` here for consistency with the `readChar` call above?

(though `DataOutput` defines both `writeChar` and `writeShort` to be implemented in the same way, so this might not matter)